apps/web/src/views/PancakeSquad/hooks/useEventInfos.tsx

```diff
@@ -31,7 +31,7 @@ const useEventInfos = ({ refreshCounter, setCallback }) => {
             } as const),
         )
 
-        const client = publicClient({ chainId: ChainId.BSC })
+        const client = getViemClients({ chainId: ChainId.BSC })
```

Comment:
Why not this to match other changes?
```suggestion
    const client = viemClients[ChainId.BSC]
```